performance.rst

```diff
@@ -165,8 +165,12 @@ For more details, see :doc:`/components/class_loader/cache_class_loader`.
 .. index::
    single: Performance; Bootstrap files
 
-Use Bootstrap Files
--------------------
+Use Bootstrap Files (For < PHP 7)
+---------------------------------
+
+.. note::
+
+    For PHP 7 and higher versions it is not necessary. It is deprecated for Symfony 4.0.
```

Comment:
I believe we used `caution` directives for deprecations in the docs. I think we should also provide a little bit of context to this message, like this:

```rst
.. caution::

    The performance optimalizations in PHP 7 removed any overhead. As such,
    using the bootstrap file is discouraged when using PHP 7+ and was
    deprecated in Symfony 4.0. 
```